website/blog/2020-03-26-version-0.62.md

```diff
@@ -10,6 +10,8 @@ tags: [announcement, release]
 
 Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
 
+<!--alex ignore master-->
```

Comment:
Should we just be changing `master` everywhere to something else? Has that conversation happened elsewhere?

Reply:
`master` is there is a reference to the repository branch. Maybe we can change it to `main` or something else, but I don't have sufficient permission for that operation. AFAIK there was no conversation earlier on this topic in website repository.